pyproject.toml

```diff
@@ -42,3 +42,45 @@ ignore = [
 
 [tool.ruff.lint.flake8-quotes]
 avoid-escape = false
+
+[tool.ruff.format]
+exclude = [
+    "python/ray/core/src/ray/gcs/",
+    "python/ray/serve/_private/benchmarks/streaming/_grpc/test_server_pb2_grpc.py",
+    "doc/external/",
+    # TODO: Remove the following exclusions
```

Comment:
why excluding those files that are covered by black now?

Reply:
I found that's because the newest version of `black` formats those files differently.

The following are the outputs for `black` verision `v25.1.0`

![image](https://github.com/user-attachments/assets/0583116f-abd6-4870-8779-3f2c7be651ff)

Reply:
I found that the output for `black` `v25.1.0` and `ruff format` are the same.